opfunu/cec_based/cec2019.py

```diff
@@ -146,14 +149,15 @@ def __init__(self, ndim=None, bounds=None, f_shift="shift_data_3", f_bias=1.):
         self.make_support_data_path("data_2019")
         self.f_shift = self.check_shift_data(f_shift)[:self.ndim]
         self.f_bias = f_bias
-        self.f_global = f_bias
+        # f_global calculated by verifying the cec2019 C value for f(x*) where x*==f_shift
+        self.f_global = 12.712062001703194 + self.f_bias
         self.x_global = self.f_shift
         self.paras = {"f_shift": self.f_shift, "f_bias": self.f_bias}
 
     def evaluate(self, x, *args):
         self.n_fe += 1
         self.check_solution(x, self.dim_max, self.dim_supported)
-        return operator.lennard_jones_minimum_energy_cluster_func(x) + self.f_bias
+        return operator.lennard_jones_func(x) + self.f_bias
```

Comment:
The previous function produced the same values except for the penalization, this version is faster by a little.